### DISCUSSION TITLE
New chaos experiment for kubernetes apiserver

### DISCUSSION THREAD

**Discussion by umamukkara** [litmuschaos/litmus#4515] [General]
Kubenetes API server is one of the key services on the K8s. Users may be interested in introducing chaos into apiserver to validate the overall stability of the system. 

This can be in the `generic` chart.

**Comment by ksatchit** [comment]
Update: 

- With the introduction of the target Host/IP filters in the network chaos experiments, microservices can be denied access to kube-api server selectively, by passing the kube-apiserver FQDN (service name) as value to the TARGET_HOSTs env on the network loss experiment. This works on both managed k8s services (gke,eks,aks et al) as well as self-managed clusters (kubeadm, kops, rancher etc..,)
